smriprep/utils/bids.py

```diff
@@ -50,81 +51,6 @@ def get_outputnode_spec():
     return fields
 
 
-def predict_derivatives(subject_id, output_spaces, freesurfer):
```

Comment:
This didn't seem to be used anywhere, and deleting this freed me up to change `collect_derivatives` to use `BIDSLayout` and use lists as "or" instead of "and" in `io_spec.json`.